### ISSUE TITLE
Finds "import.meta.vitest" but defines import as undefined

### ISSUE THREAD

**Issue by MathiasWP** [guybedford/es-module-lexer#152]
Is this expected behaviour?

**Comment by guybedford** [comment]
Yes, this is by design - `import.meta` is detected as having a `import.d === -2` value, and offsets are provided, but parsing of properties must be handled manually due to es-module-lexer not including an expression or property parser.
